packages/coordinator/tests/services/translation/request/dispense/dispense-notification.spec.ts

```diff
@@ -18,7 +18,7 @@ import {
 import {ElementCompact} from "xml-js"
 import pino from "pino"
 import {OrganisationTypeCode} from "../../../../../src/services/translation/common/organizationTypeCode"
-import {NonDispensingReason} from "../../../../../../models/hl7-v3"
+import {NonDispensingReason, PertinentInformation2NonDispensing} from "../../../../../../models/hl7-v3"
```

Comment:
Consider importing only the hl7v3 module, instead of each class, to reduce the number of imports required, e.g.

```ts
import {hl7v3} from "@models"

...
const x = new hl7v3.NonDispensingReason(...)
```

Reply:
Unnecessary imports have been removed